apollo-server/v4/integration-testing/src/server.ts

```diff
@@ -0,0 +1,41 @@
+import { ApolloServer } from '@apollo/server';
+import { startStandaloneServer } from '@apollo/server/standalone';
+
+// Schema definition
+export const typeDefs = `
+  #graphql
+  type Query {
+    hello(name: String): String!
+  }
+`;
+
+// Resolver map
+export const resolvers = {
+  Query: {
+    hello: (_, { name }) => `Hello ${name}!`,
+  },
+};
+
+// This function will create a new server Apollo Server instance
+export const createApolloServer = async (options = { port: 4000 }) => {
+  const server = new ApolloServer({
+    typeDefs,
+    resolvers,
+  });
+
+  const { url } = await await startStandaloneServer(server, { listen: options });
+
+  if (process.env.NODE_ENV !== 'test') {
```

Comment:
Maybe just move this to the if block at the bottom of the file

Reply:
Will do!